Tests/PackageGraphPerformanceTests/Inputs/SourceKitten.json

```diff
@@ -0,0 +1,204 @@
+{
+  "constraints": [
+    {
+      "identifier": "https://github.com/Carthage/Commandant.git",
+      "requirement": [
+        "0.11.3",
+        "0.11.9223372036854775807"
+      ]
+    },
+    {
+      "identifier": "https://github.com/drmohundro/SWXMLHash.git",
+      "requirement": [
+        "3.0.3",
+        "3.0.4"
+      ]
+    },
+    {
+      "identifier": "https://github.com/jpsim/Yams.git",
+      "requirement": [
+        "0.1.2",
+        "0.1.3"
+      ]
+    },
+    {
+      "identifier": "https://github.com/norio-nomura/Clang_C.git",
+      "requirement": [
+        "1.0.0",
+        "1.9223372036854775807.9223372036854775807"
+      ]
+    },
+    {
+      "identifier": "https://github.com/norio-nomura/SourceKit.git",
+      "requirement": [
+        "1.0.0",
+        "1.9223372036854775807.9223372036854775807"
```

Comment:
Is it possible to get these to be the abstract requirements, not the ones where we do the horrible max int thing?